models/brats_mri_axial_slices_generative_diffusion/configs/train_diffusion.json

```diff
@@ -0,0 +1,174 @@
+                    32,
+                    1
+                ]
+            },
+            {
+                "_target_": "RandSpatialCropSamplesd",
+                "keys": "image",
+                "random_size": false,
+                "roi_size": "$[@train_patch_size[0], @train_patch_size[1], 1]",
+                "num_samples": "@train_batch_size_slice"
+            },
+            {
+                "_target_": "SqueezeDimd",
+                "keys": "image",
+                "dim": 3
+            }
+        ],
+        "preprocessing": {
+            "_target_": "Compose",
+            "transforms": "$@preprocessing_transforms + @train#crop_transforms"
+        },
+        "dataset": {
+            "_target_": "monai.apps.DecathlonDataset",
+            "root_dir": "@dataset_dir",
+            "task": "Task01_BrainTumour",
+            "section": "training",
+            "cache_rate": 1.0,
+            "num_workers": 8,
+            "download": "@download_brats",
+            "transform": "@train#preprocessing"
+        },
+        "dataloader": {
+            "_target_": "DataLoader",
+            "dataset": "@train#dataset",
+            "batch_size": "@train_batch_size_img",
+            "shuffle": true,
+            "num_workers": 0
+        },
+        "handlers": [
+            {
+                "_target_": "LrScheduleHandler",
+                "lr_scheduler": "@lr_scheduler",
+                "print_lr": true
+            },
+            {
+                "_target_": "CheckpointSaver",
+                "save_dir": "@ckpt_dir",
+                "save_dict": {
+                    "model": "@diffusion"
+                },
+                "save_interval": 0,
+                "save_final": true,
+                "epoch_level": true,
+                "final_filename": "model.pt"
+            },
+            {
+                "_target_": "StatsHandler",
+                "tag_name": "train_diffusion_loss",
+                "output_transform": "$lambda x: monai.handlers.from_engine(['loss'], first=True)(x)"
+            },
+            {
+                "_target_": "TensorBoardStatsHandler",
+                "log_dir": "@tf_dir",
+                "tag_name": "train_diffusion_loss",
+                "output_transform": "$lambda x: monai.handlers.from_engine(['loss'], first=True)(x)"
+            }
+        ],
+        "trainer": {
+            "_target_": "scripts.ldm_trainer.LDMTrainer",
+            "device": "@device",
+            "max_epochs": 1000,
+            "train_data_loader": "@train#dataloader",
+            "network": "@diffusion",
+            "autoencoder_model": "@autoencoder",
+            "optimizer": "@optimizer",
+            "loss_function": "@loss",
+            "latent_shape": "@latent_shape",
+            "inferer": "@inferer",
+            "key_train_metric": "$None",
+            "train_handlers": "@train#handlers"
+        }
+    },
+    "initialize": [
+        "$monai.utils.set_determinism(seed=0)"
```

Comment:
I forgot to ask when reviewing the 3d bundle. Do these two bundles support deterministic training?

Reply:
Yes

Reply:
Yes, it is deterministic